modules/deploy-red-hat-openshift-container-storage.adoc

```diff
@@ -9,57 +9,57 @@
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/deploying_openshift_data_foundation_on_vmware_vsphere[Deploying OpenShift Data Foundation 4.13 on VMware vSphere]
 
 |Instructions on deploying {rh-storage} using Amazon Web Services for local or cloud storage
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/deploying_openshift_data_foundation_using_amazon_web_services[Deploying OpenShift Data Foundation 4.12 using Amazon Web Services]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/deploying_openshift_data_foundation_using_amazon_web_services[Deploying OpenShift Data Foundation 4.13 using Amazon Web Services]
 
 |Instructions on deploying and managing {rh-storage} on existing Red Hat {product-title} Google Cloud clusters
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/deploying_and_managing_openshift_data_foundation_using_google_cloud[Deploying and managing OpenShift Data Foundation 4.12 using Google Cloud]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/deploying_and_managing_openshift_data_foundation_using_google_cloud[Deploying and managing OpenShift Data Foundation 4.13 using Google Cloud]
 
 |Instructions on deploying and managing {rh-storage} on existing Red Hat {product-title} Azure clusters
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/deploying_openshift_data_foundation_using_microsoft_azure/index[Deploying and managing OpenShift Data Foundation 4.12 using Microsoft Azure]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/deploying_openshift_data_foundation_using_microsoft_azure/index[Deploying and managing OpenShift Data Foundation 4.13 using Microsoft Azure]
 
 |Instructions on deploying {rh-storage} to use local storage on IBM Power infrastructure
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html-single/deploying_openshift_data_foundation_using_ibm_power/index[Deploying OpenShift Data Foundation on IBM Power]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html-single/deploying_openshift_data_foundation_using_ibm_power/index[Deploying OpenShift Data Foundation on IBM Power]
 
 |Instructions on deploying {rh-storage} to use local storage on IBM Z infrastructure
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/deploying_openshift_data_foundation_using_ibm_z_infrastructure/index[Deploying OpenShift Data Foundation on IBM Z infrastructure]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/deploying_openshift_data_foundation_using_ibm_z_infrastructure/index[Deploying OpenShift Data Foundation on IBM Z infrastructure]
```

Comment:
This link opens a 404.

Reply:
@tamireran. Can you comment on this? Is there no 4.13 version of this doc. The link to the 4.12 version does work.